Compiler/FrontEnd/ExpressionDump.mo

```diff
@@ -1736,6 +1736,14 @@ algorithm
   end match;
 end printSubscript;
 
+public function printSubscriptStrTpl
```

Comment:
@wibraun Why do you introduce this function? It is not used and the description is wrong.